lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorService.java

```diff
@@ -306,6 +307,11 @@ public void allocate(AllocateRequest request, StreamObserver<LongRunning.Operati
                 try (var tx = TransactionHandle.create(storage)) {
                     final var volumes = prepareVolumeRequests(request.getVolumesList(), tx);
 
+                    var clusterType = switch (request.getClusterType()) {
+                        case USER, UNSPECIFIED, UNRECOGNIZED -> ClusterRegistry.ClusterType.User;
```

Comment:
plz, don't do this ;)
`UNRECOGNISED` - is an error, always
`UNSPECIFIED` - is a logical error